### PR TITLE
website: Update toggle switch documentation & demos

### PR DESCRIPTION
**Pull request by FlyersPh9**
- Add toggle switch demos for the different options.
- Update documentation, include adding hyperlinks to other mentioned components.

### CODE REVIEW

**Comment by FlyersPh9** [comment]
I noticed the toggle switch props only list `label`, `labelPosition`, and `setFocus`.  Why aren't `icon`, `size` showing?

**Comment by gretanausedaite** [comment]
> I noticed the toggle switch props only list `label`, `labelPosition`, and `setFocus`. Why aren't `icon`, `size` showing?

Probably a bug in how we get props.

File: apps/website/src/examples/ToggleSwitch.icon.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { ToggleSwitch, Flex } from '@itwin/itwinui-react';
+import { SvgCheckmark } from '@itwin/itwinui-icons-react';
+
+export default () => {
+  return (
+    <Flex flexDirection='column' alignItems='flex-start'>
+      <ToggleSwitch
+        onChange={() => {}}
```
Comment (gretanausedaite) [inline apps/website/src/examples/ToggleSwitch.icon.tsx:13]: You can remove these empty onChanges

**Review by gretanausedaite — APPROVED** [review]
Docs looks good

File: apps/website/src/examples/ToggleSwitch.icon.tsx (hunk 1 above)
Comment (FlyersPh9) [reply apps/website/src/examples/ToggleSwitch.icon.tsx]: Done!

**Pull request merged**